ptx/sec_deriv_interpret.ptx

```diff
@@ -1137,10 +1141,12 @@
         </webwork>
       </exercise>
 
+			<p><pagebreak-latex/></p>
```

Comment:
There are three consecutive pagebreaks here. Shouldn't there be at most one?

I think you need to remove the ones on lines 1127 and 1131.

You could also try editing the .tex file, and adding `\nopagebreak[4]` at the end of the introduction to the exercise group (immediately after the text "Identify which function is the derivative of the other.").

Eventually this will be added automatically, but there is still some ongoing discussion about it.

Reply:
I didn't intend for all those to stay. I think I was trying to put them in as many places as possible to see what would actually have the desired effect.